packages/typespec-test/test/openai_generic/generated/typespec-ts/src/rest/models.ts

```diff
@@ -6,11 +6,7 @@ export interface CreateTranscriptionRequest {
    * The audio file object (not file name) to transcribe, in one of these formats: flac, mp3, mp4,
    * mpeg, mpga, m4a, ogg, wav, or webm.
    */
-  file:
-    | string
-    | Uint8Array
-    | ReadableStream<Uint8Array>
-    | NodeJS.ReadableStream;
+  file: string;
```

Comment:
Could you help to explain why we change this?

Reply:
For RLC layer, we only generate 
  ```
 string
    | Uint8Array
    | ReadableStream<Uint8Array>
    | NodeJS.ReadableStream;
``` for binary bytes, if it's other encoded bytes, we will just treat it as normal string, because of the tcgc lint rule for bytes in multipart/form-data request, we have to remove the encode decorator.

Reply:
Maybe the right solution for RLC here is to figure out a way to suppress the linter rule in TCGC, instead of changing the typespec ?

Reply:
This will come back to be a File with Mary's PR, is that correct?

Reply:
yes, @MaryGao can you help verify if your calculation will not rely on any encode information on bytes?

Reply:
yes, the change is back now: https://github.com/Azure/autorest.typescript/pull/2258/files#diff-a83560ba5fcefbeffbde9968ac475fc716fdadef3cf6702aa15c166960e87170